params.ts

```diff
@@ -117,6 +117,12 @@ export async function getParameters(): Promise<Params> {
   const includeTags = parseTags(core.getInput('include-tags', { required: false }))
   const excludeTags = parseTags(core.getInput('exclude-tags', { required: false }))
 
+  const appFilePath = core.getInput('app-file', { required: false })
+  const appBinaryId = core.getInput('app-binary-id', { required: false })
+  if ((appFilePath !== "") !== (appBinaryId !== "")) {
+    throw new Error("Either app-file or app-binary-id must be used")
+  }
```

Comment:
```suggestion
  if (appFilePath === '' && appBinaryId === '') {
    throw new Error("Either app-file or app-binary-id must be used")
  }
```

Nit: can we write it this way? Finding it a bit hard to read :)

Reply:
I agree, but your suggestion is logically different. What I did was a **xor** operation, your suggestion isn't.

```suggestion
  if ((appFilePath === '' && appBinaryId === '') || (appFilePath !== '' && appBinaryId !== '')) {
    throw new Error("Either app-file or app-binary-id must be used")
  }
```

Do you think this one is better?

Reply:
Got it, it should clearly be XOR here -- thanks Igor!